kitchen.yml

```diff
@@ -54,11 +65,22 @@ platforms:
       image: netmanagers/salt-2017.7-py2:debian-8
   - name: ubuntu-1604-2017-7-py2
     driver:
-      image: netmanagers/salt-2017.7-py2:ubuntu-1604
-  # - name: centos-6-2017-7
+      image: netmanagers/salt-2017.7-py2:ubuntu-16.04
+  # Unavailable at the current time
+  # - name: centos-7-2017-7-py2
```

Comment:
This is a current distro with an old salt version. Do we want this combination? Is yes, I should add the image to the builder. If no, can you remove these comments? :yum:

Reply:
Ref: https://github.com/netmanagers/salt-image-builder/pull/1.  This was a placeholder before you managed to resolve `centos-6`!  However, `centos-6` will need some effort to get working here.  So is this still valid or not?  In the meantime, I've removed the comment.

Reply:
Added an issue #104 to fix the services tests. I think this is OK for now.